app/javascript/crayons/MentionAutocompleteTextArea/__stories__/mention-autocomplete.md

```diff
@@ -1,8 +1,12 @@
 ## Mention autocomplete
```

Comment:
did this slip in?

Reply:
It did, from a pre-commit hook fixed some linter issues. Just reverted the changes 🙂